mne/io/brainvision/brainvision.py

```diff
@@ -256,21 +256,25 @@ def _read_annotations_brainvision(fname, sfreq='auto'):
     annotations : instance of Annotations
         The annotations present in the file.
     """
-    onset, duration, description, date_str = _read_vmrk(fname)
-    orig_time = _str_to_meas_date(date_str)
-
-    if sfreq == 'auto':
-        vhdr_fname = op.splitext(fname)[0] + '.vhdr'
-        logger.info("Finding 'sfreq' from header file: %s" % vhdr_fname)
-        _, _, _, info = _aux_vhdr_info(vhdr_fname)
-        sfreq = info['sfreq']
-
-    onset = np.array(onset, dtype=float) / sfreq
-    duration = np.array(duration, dtype=float) / sfreq
-    annotations = Annotations(onset=onset, duration=duration,
-                              description=description,
-                              orig_time=orig_time)
-
+    if np.size(_read_vmrk(fname)) > 0:
+        onset, duration, description, date_str = _read_vmrk(fname)
```

Comment:
you should not read the vmrk twice. Read it above the if. Then do

if onset.size > 0:

etc...

Reply:
It is reading the vmrk which causes the ValueError `not enough values to unpack (expected 4, got 0)` if the condition is not met.

Reply:
```
markers = _read_vmrk(fname)
if len(markers) > 0:
     onset, duration, description, date_str = out
     ...
else:
```

ok?

Reply:
Yes!